pkg/operator/ceph/nfs/controller.go

```diff
@@ -204,6 +205,15 @@ func (r *ReconcileCephNFS) reconcile(request reconcile.Request) (reconcile.Resul
 	}
 	r.clusterInfo.CephVersion = currentCephVersion
 
+	if r.clusterInfo.CephVersion.IsAtLeastPacific() {
+		if r.clusterInfo.CephVersion.IsAtLeast(version.CephVersion{Major: 16, Minor: 2, Extra: 6}) {
```

Comment:
Please add `version.CephVersion{Major: 16, Minor: 2, Extra: 6}` as a variable and comment it to explain its purpose.

Reply:
[here](https://github.com/rook/rook/pull/8501/files#diff-4e2a2760e2988f8ea46dfdf0cbf1799d3f2591fba745b8984d82f1c84382cebeR54)